group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupMetadataManager.java

```diff
@@ -3284,7 +3285,7 @@ private boolean maybeUpdateRegularExpressions(
      * @param log           The log instance.
      * @param time          The time instance.
      * @param image         The metadata image to use for listing the topics.
-     * @param authorizer    The authorizer.
+     * @param authorizerPlugin    The authorizer.
```

Comment:
How about update to `The authorizer is wrapped in a {@link org.apache.kafka.common.internals.Plugin}`?

Reply:
I think it's a bit overkill. This field was an `Optional<Authorizer>` and we just said `The authorizer`.

Reply:
Thanks, make sense to me.